### PR TITLE
Rekognition api

### PR DESCRIPTION
**Pull request by dgbright**
Updated the readme to discuss more app setup (running commands to get the app to run, but also getting the API keys).

Added a class for handing the Rekognition API calls (scan a predetermined video; pull that response).
Added a class for handling the Rekognition response data structure.

Updated Video Recording screen to interface with rekognition api.
Added a Video Response UI screen to review the results of the selected scan job.

[Open Video screen.docx](https://github.com/umgc/fall2023/files/12851528/Open.Video.screen.docx)

### CODE REVIEW

File: cogniopenapp/pubspec.yaml [hunk 1]
```diff
@@ -35,10 +35,12 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  aws_rekognition_api: ^2.0.0
+  flutter_dotenv: ^5.1.0
+  flutter_speed_dial: ^7.0.0
   chat_gpt_flutter: ^1.0.15
   audioplayers: ^5.2.0
   local_auth: ^2.1.7
-  flutter_dotenv: ^5.1.0
```
Comment (GabrielQueenieGomes) [inline cogniopenapp/pubspec.yaml:41]: is there a way i can correct the red areas?

**Review by GabrielQueenieGomes — COMMENTED** [review]
Do the red areas indicate that there are errors to fix? if so, how can i help you all in correcting them?

**Review by zcappella — APPROVED** [review]

**Comment by zcappella** [comment]
Did you merge camera screen branch into this one?

**Review by BenjaminSutter — APPROVED** [review]
Good enough of a foundation for future work

**Pull request merged**